src/controller_examples/rabbitmq_controller/spec/reconciler.rs

```diff
@@ -28,6 +28,8 @@ impl Reconciler<RabbitmqClusterView> for RabbitmqReconciler {
     type T = RabbitmqReconcileState;
     type I = ();
     type O = ();
+    type S = ();
```

Comment:
We probably should find better names for I, O and S. Any ideas?

Reply:
Of course. But this may make some lines look lengthy :-((

Reply:
That's totally fine